qiskit/transpiler/passes/layout/noise_adaptive_layout.py

```diff
@@ -48,10 +48,10 @@ class NoiseAdaptiveLayout(AnalysisPass):
      participate in any CNOT), map them to any available
      hardware qubit.
 
-     Notes:
-         even though a `layout` is not strictly a property of the DAG,
-         in the transpiler architecture it is best passed around between passes
-         by being set in `property_set`.
+    Notes:
+     even though a `layout` is not strictly a property of the DAG,
+     in the transpiler architecture it is best passed around between passes
+     by being set in `property_set`.
```

Comment:
Something's gone wrong with the indentation here (probably with the entire docstring).